scripts/install/roles/vagrant/tasks/main.yml

```diff
@@ -45,7 +45,7 @@
     which vagrant 2>&1 >/dev/null
   register: vagrant_installed
 
-- fail: "Vagrant must be installed"
+- fail: msg="Vagrant must be installed"
```

Comment:
Can we use the YAML object format as everywhere else on this file instead of the custom ansible stuff which works only when the parser is not broken?